index.js

```diff
@@ -333,9 +333,9 @@ app.post('/api/reactions/:id', cors(corsOptions), async (req, res) => { // react
 
 app.get('/auth/begin', (req, res) => {
     if (req.get('host') == 'localhost:8081') {
-        res.redirect(`https://fluffyscratch.hampton.pw/auth/getKeys/v2?redirect=bG9jYWxob3N0OjgwODEvYXV0aC9oYW5kbGU=`)
+        res.redirect(`https://auth.itinerary.eu.org/auth/?redirect=bG9jYWxob3N0OjgwODEvYXV0aC9oYW5kbGU=&name=Ocular`)
```

Comment:
For the localhost base64 encoded URL, you should include the "http" prefix as Scratch Auth assumes "https" when you don't specify a protocol, which wouldn't work on localhost. The correct encoded URL is therefore `aHR0cDovL2xvY2FsaG9zdDo4MDgxL2F1dGgvaGFuZGxl`